### PR TITLE
[CI/Build] Refactor image test assets

### PR DESCRIPTION
**Pull request by DarkLight1337**
This PR makes it a bit more convenient to use test images by organizing related methods into a new class (`ImageAsset`).

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -51,6 +43,63 @@ def _read_prompts(filename: str) -> List[str]:
...
+
+
+class _ImageAssets(UserList[ImageAsset]):
+
+    def __init__(self) -> None:
+        super().__init__(
+            [ImageAsset("stop_sign"),
+             ImageAsset("cherry_blossom")])
+
+    def prompts(self, prompts: _ImageAssetPrompts) -> List[str]:
```
Comment (xwjiang2010) [inline tests/conftest.py:89]: nit: `set_prompts` to be clear.

File: tests/conftest.py [hunk 2]
```diff
@@ -51,6 +43,63 @@ def _read_prompts(filename: str) -> List[str]:
...
+        """
+        Convenience method to define the prompt for each test image.
```
Comment (xwjiang2010) [inline tests/conftest.py:91]: nit: convenient method.

File: tests/conftest.py (hunk 1 above)
Comment (xwjiang2010) [reply tests/conftest.py:89]: hmm, actually this method is set and get in one go :)

**Review by xwjiang2010 — APPROVED** [review]

**Comment by ywang96** [comment]
@DarkLight1337 Do we still want this in? IMO now that we're moving towards the direction of #5852, we should leave this refactor for later. WDYT?

**Comment by DarkLight1337** [comment]
Even after #5852, this PR would still simplify the code. Since #5852 will not make it into the upcoming release, let's merge this first.

**Review by ywang96 — APPROVED** [review]

**Pull request merged**